roles/docker_image_build/templates/base/etc/system/local/server.conf

```diff
@@ -0,0 +1,4 @@
+{% if migrate_kvstore is defined %}
```

Comment:
should this conditional also check the splunk version?

Reply:
This is only needed on v8, but it gets removed automatically as part of the built-in splunk migration process.